server/templates/deepseek_fim.yaml

```diff
@@ -1,13 +1,15 @@
+# Since deepseek_fim_client.py makes requests with a prefix/suffix directly, `prompt` and `args` are not utilized.
```

Comment:
Note: the prompt and args are there for consistency purposes. I do use them (though perhaps I should change the FiM function to just use the prefix directly at this point).

Reply:
Sure, feel free to update the comments here to reflect current status?